### ISSUE TITLE
Yash Navlakha

### ISSUE THREAD

**Issue by aflies1** [promazo/Sourcing#176]

**Comment by aflies1** [comment]
Career? 
Previously worked as an intern consultant, has been in talks with BCG and Goldman
Has had a few companies reach out in terms of research as well
Would like to be client facing consultant who has data skills

Python?
Has been studying python for 6 years, has taken some courses in the US as well
Ranks as an 8.5, does not know everything but how to resolve it
Has used Python in both his internships
Did use NLP similar to what we are going to need to do with our client

Larget datasets?
Has developed python skipts for 3.2 m rows
Large volume he has worked with millions of rows of data in those roles

ML?
Has taken courses
Has done NLP, this was client work. 
Has built models to predict and classify, etc

Schedule?
Would have 3 classes (monday morning and afternoon, wednesday afternoon he would have class as well. Would have 2 hours daily. Only monday would be difficult, his class on wednesday is 6-8.
He does have plans to travel plans to go back to India, last semester he did work 20-30 hours per week from India. Would be Mid- Dec - Early Jan
OK to stay in the US if needed.

**Comment by aflies1** [comment]
Resume; https://drive.google.com/drive/folders/1nWwkVJCtvCkvgcx3xfonrgqEFwSh67dO

**Comment by ldaudet** [comment]
Yash Navlakha

During his Bachelor, he got to take a bunch of courses outside of his major, and he decided to get a ton of CS course. He also realized that electronc wasn't for him.

His family has a business back in india, and he used to work on Excel to help with it. He got exposed to Tableau trhough it. Having to create clear visualisations for his family, he realized that this was an actual interest of his, more so than electronics. 

After coming to the USA in spring 2021, he started to work as a data analyst intern at the Business Inteligence group and started to apply his ML and NLP skills. He self taught himself a lot of the skills that he used there during his internship there. That included BERT, though he ended up not using it for the project because of his computational requirements. 

He worked in Jupyter notebook and google collab before. 

He graduates in 3 months though, so he ll need something full time after that date. Never mind, he is graduating in May because he needs more time to finish some projects for school. 

He got to use XGBoost in the past in a project to convert hand written digits. The same project also made use of CNN, so he got exposed to NN in the past. 

He is also trying to learn Business Intelligence, like SQL and tableau, creating dashboard. 

He asked me for a timeline to get back to him, because he has other offers he needs to get back too, and I told him Monday or Tuesday of next week after I get a chance to check in with Adam.


TL;DR: I think he might be better suited for the internal algo or USAA in terms of skills. With his graduation in May, I think that USAA is off the table, leaving the internal algo. He could also be a good tech lead at prescient with a more junior profile.
